src/PowerShellEditorServices/Services/PowerShell/Host/ConsoleColorProxy.cs

```diff
@@ -0,0 +1,153 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+/*
```

Comment:
TODO?

Reply:
From Rob: we need to go implement this with @SteveL-MSFT and @daxian-dbw's help because it's an "obscure" interface.